### PR TITLE
[13.0][MIG] stock_picking_line_sequence

### PR DESCRIPTION
**Pull request by RaulOForgeFlow**
@LoisRForgeFlow @JordiMForgeFlow @FerranCforgeFlow

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: stock_picking_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Stock picking lines with sequence number",
+    "summary": "Manages the order of stock moves by displaying its sequence",
+    "version": "13.0.1.1.1",
```
Comment (sebalix) [inline stock_picking_line_sequence/__manifest__.py]: Didn't spot this, the version should be reset:
```suggestion
    "version": "13.0.1.0.0",
```

Comment (RaulOForgeFlow) [reply stock_picking_line_sequence/__manifest__.py]: Good catch. Thank you!

**Review by sebalix — APPROVED** [review]

**Review by hparfr — APPROVED** [review]
LGTM

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**